Cargo.toml

```diff
@@ -25,4 +25,5 @@ members = [
     "scripts/test-projects",
     "scripts/check-docs",
     "tools/fuels-abi-cli",
+    "scripts/test-projects-workspace"
```

Comment:
I would not change the script name. `test-projects` seems ok even though we use the `workspace` way of building our projects.

Reply:
Agreed

Reply:
This name is temporary